scripts/maintenance.awk

```diff
@@ -0,0 +1,14 @@
+## Copyright (C) 1996-2019 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+#
+# This script performs easily automated code updates for
+# removal or renaming of deprecated/obsolete syntax.
+# It is run by maintenance script over .h, .c, .cc, and .cci files
```

Comment:
Please mention that the script is only applied to git-tracked files (i.e. generated files are not covered).

Reply:
Done